jumpstarter/states.py

```diff
@@ -34,6 +34,34 @@
 NestedState.separator = "↦"
 
 
+class Transition(str, Enum):
+    def _generate_next_value_(name: str, *args) -> str:
+        name = name.replace("__", NestedState.separator)
+        return name.lower()
+
+    initialize = auto()
+    pause = auto()
+    recover = auto()
+    report_error = auto()
+    report_problem = auto()
+    report_warning = auto()
+    restart = auto()
+    restarting__starting = auto()
+    restarting__stopping = auto()
+    resume = auto()
+    start = auto()
+    stop = auto()
+
+
+class Event(str, Enum):
```

Comment:
I think we're using the wrong terminology here.
It could be because my design isn't 100% correct as of yet.

Reply:
> 
> 
> I think we should rename that enum to `ActorLifecycleEvent`.
> That would be more helpful.